docker-compose.yml

```diff
@@ -933,7 +931,10 @@ services:
     environment:
       <<: *ccache
     volumes: *conda-volumes
-    command: *python-conda-command
+    command:
+      ["/arrow/ci/scripts/cpp_build.sh /arrow /build &&
+        /arrow/ci/scripts/python_build.sh /arrow /build true &&
```

Comment:
Is there a point in building docs outside of the dedicated `ubuntu-docs` build?

Reply:
Yes. The `ubuntu-docs` build can take more than 90 minutes so we shouldn't run that on all PRs. Instead we can build language specific docs for PRs and execute the full docs built on the master branch.